<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>openXLS</class>
 <widget class="QDialog" name="openXLS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>117</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open XLS\XLSX File</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Source</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sourcelineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="openButton">
       <property name="text">
        <string>&amp;Open</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="filegroupBox">
       <property name="title">
        <string>File Format</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="headercheckBox">
          <property name="text">
           <string>header</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="rownamecheckBox">
          <property name="text">
           <string>Row Names</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="okButton">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>&amp;OK</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>&amp;Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>openXLS</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>openXLS</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
